controller_manager/src/controller_manager.cpp

```diff
@@ -1454,17 +1423,17 @@ void ControllerManager::activate_controllers(
   const std::vector<ControllerSpec> & rt_controller_list,
   const std::vector<std::string> controllers_to_activate)
 {
-  for (const auto & request : controllers_to_activate)
+  for (const auto & controller_name : controllers_to_activate)
```

Comment:
You have a problem here as '''controller_name''' is later defined in line 1440.

Reply:
I think this is the new shadowing errors that we have enabled

Reply:
@bmagyar sabotaged the PR since he merged uncarefully master into PR.

Reply:
I'm planning to take a new online course "Maintainership for beginners" very soon! :D